src/content/learn/sharing-state-between-components.md

```diff
@@ -1,31 +1,34 @@
 ---
-title: Sharing State Between Components
+title: Berbagi State Antar Komponen
+
 ---
 
 <Intro>
 
-Sometimes, you want the state of two components to always change together. To do it, remove state from both of them, move it to their closest common parent, and then pass it down to them via props. This is known as *lifting state up,* and it's one of the most common things you will do writing React code.
+Terkadang, Anda ingin 2 komponen selalu berubah secara bersamaan. Untuk melakukannya, hapus *state* dari kedua komponen, pindahkan ke komponen induk terdekat, dan kemudian oper ke komponen tersebut melalui *props*. Ini dikenal sebagai mengangkat *state* keatas, dan ini adalah salah satu hal yang paling umum yang akan Anda lakukan saat menulis kode React.
 
 </Intro>
 
 <YouWillLearn>
 
-- How to share state between components by lifting it up
-- What are controlled and uncontrolled components
+- Bagaimana cara berbagi state antar komponen dengan *lifting state up*
+- Apa itu komponen terkendali dan tidak terkendali
```

Comment:
According to the [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium) the term `controlled component` shouldn't be translated, but that was in the past. Should we change how we translate this? @mhaidarhanif @r17x @zainfathoni

Reply:
Yes, I think we should translate it. "Komponen terkendali" is already explicit enough and easily understood.

Reply:
I would prefer "tak terkendali" to "tidak terkendali", though. It's more concise and simpler.

Reply:
@resir014 @zainfathoni Okay how about:

- `komponen terkendali (*controlled component*)`
- `komponen tak erkendali (*uncontrolled component*)`

As usual, the English words are introduced once in the beginning. Then after that just use the translated version.

Reply:
it is better with `tak terkendali` mas